lib/github_service/commands/cross_repo_test.rb

```diff
@@ -246,6 +268,18 @@ def create_cross_repo_test_pull_request
                                           "[BOT] Cross repo test for #{issue.fq_repo_name}##{issue.number}", pr_desc)
       end
 
+      def self.fetch_manageiq_release_repo_groups
+        require 'net/http'
+        require 'uri'
+
+        uri = URI.parse("https://raw.githubusercontent.com/ManageIQ/manageiq-release/master/config/repo_groups.yml")
```

Comment:
To make it also usually in manageiq-release, let's name it repos.groups.yml

Reply:
:+1: